team_test.go

```diff
@@ -318,13 +297,13 @@ func TestTeamMembers(t *testing.T) {
 			Id: id4,
 		},
 	}
-	resp, err := team.GetMembers(client, nil)
+	resp, err := team.GetMemberships(client, nil)
 	result := resp.Nodes
 	// Assert
 	autopilot.Ok(t, err)
 	autopilot.Equals(t, 3, resp.TotalCount)
-	autopilot.Equals(t, "kyle@opslevel.com", result[0].Email)
-	autopilot.Equals(t, "Matthew Brahms", result[2].Name)
+	autopilot.Equals(t, "kyle@opslevel.com", result[0].User.Email)
+	// autopilot.Equals(t, "Matthew Brahms", result[2].Name)
```

Comment:
This line stuck out to me, should it be removed?

Reply:
Yes, or maybe I'll uncomment it. I forget the background behind this but I'll resolve it either way. Good call out! 👍

Reply:
Removed ✅